sensor_board/common/pinav_parser.h

```diff
@@ -0,0 +1,90 @@
+#ifndef _PINAV_PARSER_H_
+#define _PINAV_PARSER_H_
+
+#include <stdint.h>
+
+// All flight code will be in C, but the test library is technically C++.
+// This means that all flight code must declare itself to have C linkage when
+// compiled in C++ mode.
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+// Status enum returned by parse function
+typedef enum pinav_parser_status	{	OK, 
+										NULL_OUTPUT_PTR,
+										NULL_SENTENCE_PTR,
+										UNRECOGNIZED_SENTENCE_TYPE,
+										IMPROPER_SENTENCE_LENGTH,
+										SENTENCE_FORMAT_ERROR,
+										UNKNOWN_ERROR // @TODO: remove once I'm sure this never happens
+									} pinav_parser_status_t;
+
+// This identifies one of several sentence formats described in the piNAV datasheet
+typedef enum pinav_sentence_id {NONE, GGA, LSP} pinav_sentence_id_t;
```

Comment:
Same as above, these should probably have a prefix. `NONE` in particular seems dangerous.